README.md

```diff
@@ -1,68 +1,43 @@
-# The Planetary Software Organization Technical Steering Committee
+# The Planetary Software Organization
 
-The Planetary Software Organization Technical Steering Committee (TSC) is the technical
-governing body of the Planetary Software Organization. It is described in the
-[TSC Charter][].
+The Planetary Software Organization promotes open source software in the
+planetary sciences by helping software creators and maintainers foster an
+open source community.
 
-The governing documents for the TSC are locating in the root folder of this repository.
+The Planetary Software Organization helps software creators grow
+and organize their community by providing a set of ready-to-use
+community guidelines, standards, and governance documents.  The TSC
```

Comment:
TSC is now unknown at this point?

Reply:
Thanks, this language was only halfway fixed to where I wanted it.  Now these opening paras only talk about the larger Planetary Software Organization, and 'the TSC' isn't mentioned until it is defined below.